### PR TITLE
test: Inject Sizzle and lib into phantom pages on demand

### PR DESCRIPTION
**Pull request by stefwalter**
Each time we notice it's not there, inject in again. The idea is that this fixes a bunch of races.

### CODE REVIEW

**Comment by stefwalter** [comment]
Test failures fixed by #2825

File: test/phantom-driver [hunk 1]
```diff
@@ -50,6 +50,7 @@ var sys = require('system');
 var onCheckpoint;
 var lastError;
 var waitTimeout;
+var didTimeout;
```
Comment (mvollmer) [inline test/phantom-driver]: This is clearly the right thing to do, but I am not sure if it actually changes anything.  Without this declaration, 'didTimeout' should be a implicit global variable, not a local variable in each of the functions that use it.

Comment (stefwalter) [inline test/phantom-driver]: You're right. I was confused.

**Comment by stefwalter** [comment]
Accidentally refered to this instead of #2812 from another pull request.

**Pull request closed without merging**